src/couch_prometheus/src/couch_prometheus_server.erl

```diff
@@ -23,7 +23,9 @@
 
 -export([
     scrape/0,
-    version/0
+    version/0,
+    % for testing
+    get_internal_replication_jobs_stat/0
```

Comment:
Wonder if an `-ifdef(TEST)...` directive would work? Or that would look worse as it's more verbose?

Reply:
I feel like I tried that somewhere before and it didn't work, but yeah, that's better for sure. Fixed in https://github.com/apache/couchdb/commit/7be4f02f269b183ab52950cea5c8f87a03b2eba7

Reply:
I could think of that not working in some cases, maybe when using from another application? Up to you if you feel like it's fragile, we can keep it as is.